.github/workflows/container-image.yml

```diff
@@ -177,3 +188,43 @@ jobs:
           subject-name: index.docker.io/${{ env.IMAGE_NAME }}
           subject-digest: ${{ steps.build-and-push.outputs.digest }}
           push-to-registry: false
+
+      - name: Prepare For-Container.md file
+        if: ${{ env.LATEST == 'true' }}
+        run: |
+          if [ -z "${{ env.README_FILEPATH }}" ]; then
+            file_path=$(find ./doc/ -type f -name 'For-Container.md' | head -n 1)
+            if [ -z "$file_path" ]; then
+              echo "No For-Container.md file found in the ./doc/ directory."
+              exit 1
+            fi
+            echo "No custom container README file path provided. Using default path: $file_path"
+            echo "README_FILEPATH=$file_path" >> "$GITHUB_ENV"
+          else
+            # Check if the provided file exists.
+            if [ -f "${{ env.README_FILEPATH }}" ]; then
+              echo "Using provided container README file path: ${{ env.README_FILEPATH }}"
+            else
+              echo "Provided container README file path does not exist: ${{ env.README_FILEPATH }}"
+              exit 1
+            fi
+          fi
+
+          # Check if the README file has been modified since the github.event.before reference point
+          # and write the result to the README_MODIFIED ENV variable.
+          if ! git diff --quiet --exit-code ${{ github.event.before }} "${{ env.README_FILEPATH }}"; then
+            echo "README file has been modified since the last commit."
+            echo "README_MODIFIED=true" >> "$GITHUB_ENV"
+          else
+            echo "README file has not been modified since the last commit."
+            echo "README_MODIFIED=false" >> "$GITHUB_ENV"
+          fi
```

Comment:
Does `${{ github.event.before }}` even exist for release events? Either way, I'd just cut down on complexity here and push the sync it with every release. We aren't releasing that frequently after all that I'd expect any trouble there. Or does Docker Hub actually complain if you upload the very same file again?